consensus/objs/accusations.go

```diff
@@ -0,0 +1,68 @@
+package objs
+
+import (
+	"errors"
+
+	"github.com/google/uuid"
+)
+
+type AccusationState uint8
+
+const (
+	// Created means the accusation has been identified but not yet persisted in any way.
+	Created AccusationState = 0
+	// Persisted means the accusation has been identified and persisted in the consensus DB before being processed
+	Persisted AccusationState = 1
+	// ScheduledForExecution means the accusation is persisted in consesus DB and scheduled for execution, or executing, on the task manager. An accusation should only be scheduled for execution when in the Persisted state
+	ScheduledForExecution AccusationState = 2
+	// Completed means the accusation has been executed and no action is needed. An accusation should only be completed when in the ScheduledForExecution state
+	Completed AccusationState = 3
+)
+
+type Accusation interface {
```

Comment:
Is it expected that there will be multiple implementers for an `Accusation`? If not, we can probably skip the interface, which would allow the removal of the getters and setters below.

Reply:
Makes sense, thanks!